spirv_hlsl.cpp

```diff
@@ -4739,9 +4739,25 @@ void CompilerHLSL::emit_subgroup_op(const Instruction &i)
 			inherit_expression_dependencies(id, ops[4]);
 		}
 		else if (operation == GroupOperationInclusiveScan)
-			SPIRV_CROSS_THROW("Cannot trivially implement BallotBitCount Inclusive Scan in HLSL.");
+		{
+			auto expr = join("countbits(", to_enclosed_expression(ops[4]), " & gl_SubgroupLeMask)");
```

Comment:
This is questionable when wave size is > 32. I think you'd need to sum the components like for reduce.

Reply:
Also, I'm missing tests here.

Reply:
Changes done.